cfme/services/catalogs/catalog_item.py

```diff
@@ -294,17 +297,41 @@ def edit_tags(self, tag, value):
         assert view.is_displayed
         view.flash.assert_success_message('Tag edits were successfully saved')
 
+    def remove_tag(self, tag_category, tag_name):
+        """ Remove tag from service catalog item
+            Args:
+                tag_category: Tag category
+                tag_name: Tag name
+        """
+        view = navigate_to(self, 'EditTags')
+        row = view.tag_table.row(category=tag_category, assigned_value=tag_name)
+        row[0].click()
+        view.save_button.click()
+        view = self.create_view(DetailsCatalogItemView)
+        view.flash.assert_success_message('Tag edits were successfully saved')
+        assert view.is_displayed
+
+    @property
+    def exists(self):
+        try:
+            navigate_to(self, 'Details')
+            return True
+        except CandidateNotFound:
+            return False
 
-class CatalogBundle(Updateable, Pretty, Navigatable):
+
+class CatalogBundle(CatalogItem, Navigatable):
 
     def __init__(self, name=None, description=None, display_in=None,
                  catalog=None, dialog=None, catalog_items=None, appliance=None):
-        self.name = name
-        self.description = description
-        self.display_in = display_in
-        self.catalog = catalog
-        self.dialog = dialog
         self.catalog_items = catalog_items
+        super(CatalogBundle, self).__init__(
```

Comment:
honestly, I don't like such C-like style. it doesn't increase readability even worse.